api/validator.py

```diff
@@ -10,9 +10,10 @@
 
 
 def validate_json(validator_func):
-    '''Decorator for validating submitted json using supplied validator function.
+    """Decorator for validating submitted json using supplied validator function.
```

Comment:
Pep8 requires docstrings to be double quote while other quotes are optional but should be consistent in the project